### DISCUSSION TITLE
On installation : Warning: /opt/homebrew/bin is not in your PATH.

### DISCUSSION THREAD

**Discussion by bobettebob** [Homebrew/discussions#2831] [Getting started] [answered]
I have a macMini Monterey 12.1 with M1 chip.
I installed Xcode13.2.1 from Apple.
I installed Homebrew `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
let it Downloading and installing Command Line Tools for Xcode
I did: - Run these two commands in your terminal to add Homebrew to your PATH:
    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/tourbillon119/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"
But nothing happen Sur Sur if it's installed correctly.
Thank you helping me with the installation. :)

**Comment by carlocab** [comment]
What's the output of
```
grep brew ~/.zprofile
echo $PATH
```
?

**Reply by bobettebob** [reply]
The output is : 
% grep brew ~/.zprofile
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"

and 

% echo $PATH
/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin

**Comment by gromgit** [comment, marked as answer]
If you saw `Warning: /opt/homebrew/bin is not in your PATH` during the _Homebrew installation process_, that's perfectly normal. The full message should be:
```
Warning: /opt/homebrew/bin is not in your PATH.
  Instructions on how to configure your shell for Homebrew
  can be found in the 'Next steps' section below.
```
and the _Next steps_ section tells you to add the two `brew shellenv` commands you mentioned.

However, this:
```
% grep brew ~/.zprofile
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"
```
says you ran the pair of `brew shellenv` commands _three_ times. You should probably run `open -a TextEdit ~/.zprofile` to edit that file, and remove the second and third instances.

Finally, to be sure, run `brew doctor`. If you see anything other than:
```
Your system is ready to brew.
```
then fix whatever it tells you to, then run `brew doctor` again. Rinse and repeat as necessary.

**Reply by bobettebob** [reply]
Thank you very much! :)

**Reply by robbieroo** [reply]
Super helpful, thank you.

**Reply by moemia12** [reply]
hi does 'Your system is ready to brew' mean that homebrew is in my path?

**Reply by gromgit** [reply]
> hi does 'Your system is ready to brew' mean that homebrew is in my path?

Assuming you got that from `brew doctor`, then yes. That's one of the things it checks.

**Reply by tsurajtangoe** [reply]
how to set  path  again to usr/local/bin/brew?
for mac M2

**Reply by SMillerDev** [reply]
Install it with a Rosetta enabled terminal. But all your software will run slower than native.
